tests/js/server/aql/aql-graph-traverser.js

```diff
@@ -2197,13 +2197,15 @@ function optimizeInSuite() {
       var noOptPlans = AQL_EXPLAIN(vertexQuery, bindVars, noOpt).plan;
       assertEqual(optPlans.rules, []);
       // This query cannot be optimized by traversal rule
-      assertEqual(roundCost(optPlans), roundCost(noOptPlans));
+      // we do not want to test estimatedCost or selectivityEstimate here
```

Comment:
I think it'd be better if the comment said *why* we don't want to test them: they are 
 * subject to rounding errors and other fluctuations 
 * absolute numbers for `estimatedCost` and `selectivityEstimate` are an implementation detail and meaningless for this test.